### PR TITLE
Update to 3.0.0-alpha.1

### PR DESCRIPTION
**Pull request by erick0z**
Update runtime, proxy and shim to 3.0.0-alpha.1

Fixes https://github.com/clearcontainers/packaging/issues/2

### CODE REVIEW

File: runtime/cc-runtime.spec-template [hunk 1]
```diff
@@ -21,6 +21,7 @@ Requires: cc-runtime-config
 Requires: qemu-lite
 Requires: clear-containers-image
 Requires: linux-container
+Requires: virtcontainers-pause
```
Comment (gorozco1) [inline runtime/cc-runtime.spec-template]: The intention here is to remove completely `virtcontainers-pause` package and instead `cc-runtime` package should provide `cc-runtime` binary and `pause` binary

**Review by gorozco1 — CHANGES_REQUESTED** [review]
The real fix for #4 is to provide two binaries in a package cc-runtime 

Could you push a PR only including a fix for #2 and rework the fix for #4  ?

File: runtime/debian.rules [hunk 2]
```diff
@@ -20,7 +20,4 @@ override_dh_auto_build:
 
 override_dh_auto_install:
 	mkdir -p debian/cc-runtime
-	make install DESTDIR=`pwd`/debian/cc-runtime PREFIX=/usr
-
-
-
+	cd $(GOPATH)/src/github.com/clearcontainers/runtime/ && make install DESTDIR=`pwd`/debian/cc-runtime PREFIX=/usr SYSCONFDIR=/etc
```
Comment (gorozco1) [inline runtime/debian.rules]: Why this have to be in one line?

Comment (erick0z) [reply runtime/debian.rules]: For some reason, during the ubuntu build, it does not work having it splitted in two lines. I guess is related to the makefile stuff

Comment (erick0z) [reply runtime/debian.rules]: well, when the make is performed, it changes to another directory, but in order to perform the go build, it needs to be in the gopath.

Comment (gorozco1) [reply runtime/debian.rules]: yes Makefile needs to watch them as a single line
so 

```
cd $(GOPATH)/src/github.com/clearcontainers/runtime/ ; \
   make install DESTDIR=`pwd`/debian/cc-runtime PREFIX=/usr SYSCONFDIR=/etc
```

**Review by gorozco1 — APPROVED** [review]

**Pull request merged**